source/adl-standalone/tests/runCommand.yml

```diff
@@ -0,0 +1,54 @@
+tests:
+  -
+    description: "ListCollections succeeds using runCommand"
```

Comment:
The [scope](https://docs.google.com/document/d/1eXwy-7v_7BlPpcr7emXWSJZWm4V47ZuH5lvBj-XnOmY/edit#) only stated that:

> Drivers should be able to execute a command through existing driver runCommand helper method

@jyemin: Was the intention of that line to say that each of the commands tested by explicit helper methods should also be run through `runCommand` (as is the case here), or would it be sufficient to execute a single command (perhaps `ping` if that's supported).

cc: @craiggwilson

Reply:
I single command.

Reply:
Assuming `ping` can be used, this would probably address most of my existing comments on the revised `runCommand.yml` test file. I'll leave this comment thread open pending @jstewart-mongo's response, even though the original target here is outdated.

Reply:
Ah, Craig confirmed that `ping` is supported in https://github.com/mongodb/specifications/pull/821#discussion_r451174446.